tests/integration/test_exec.py

```diff
@@ -2,11 +2,14 @@
 __license__ = "Apache-2.0"
 
 from jina import Flow, Document
-from jinahub.SUB_PACKAGE_NAME.MODULE_NAME import MyDummyExecutor
+#from jinahub.segmenter.vggish_audio_segmenter import VGGishSegmenter
+import sys
+sys.path.insert(1, '../..')
```

Comment:
this we discussed in another PR should not be there, check VideoEncoder

Reply:
I will remove them eventually, there now for local testing